test/cases/shadowrootmode/shadowrootmode.spec.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Use Case
+ * Run wcc against two custom elements, one with an open shadow root and one with a closed shadow root
+ *
+ * User Result
+ * Should return the expected HTML output based on the shadow root mode of each component
```

Comment:
Minor suggestion here to maybe get a little more technical / specific on the result here

> _Should return the expected attribute value for shadowrootmode on the template tag based on the shadow root mode of each component_